packages/backend/src/notification/notification-stream.ts

```
@@ -0,0 +1,66 @@
+import { Observable, map, merge } from 'rxjs';
+import { NotificationSummary } from './notification-hub';
+
+/**
+ * What the SSE handler writes out. `summary` updates the client's badge;
+ * `ping` exists only so an idle connection keeps flowing through proxies that
+ * would otherwise time it out. The client ignores `ping`.
+ */
+export type StreamEvent =
+  | { type: 'summary'; data: NotificationSummary }
+  | { type: 'ping'; data: '' };
```

ESLint

[error] Replace `|·{·type:·'summary';·data:·NotificationSummary·}⏎·` with `{·type:·'summary';·data:·NotificationSummary·}`

(prettier/prettier)